ssh/config.json

```diff
@@ -22,11 +22,13 @@
   "map": ["config:rw", "ssl:rw", "addons:rw", "share:rw", "backup:rw"],
   "options": {
     "authorized_keys": [],
-    "password": ""
+    "password": "",
+    "allow_tcp_forwarding": false
```

Comment:
I think somethings like this would be better:
```yaml
server:
  tcp_forwarding: false
```

Reply:
Sure, sounds good to me. I just copied the option as-is from the SSH & Web terminal add-on.

Reply:
I just noticed the `server:` key in your comment. Is the extra level of hierarchy necessary? All of the options are for the SSH server.

Reply:
yes